src/components/App.js

```diff
@@ -0,0 +1,56 @@
+import { getUser, getFolder } from "../api";
+import "./App.css";
+import NavigationBar from "./NavigationBar";
+import { useEffect, useState } from "react";
+import FolderBar from "./FolderBar";
+import CardSection from "./CardSection";
+import Footer from "./Footer";
+
+function App() {
+  const [profile, setProfile] = useState(null);
```

Comment:
유저 정보를 불러오는 것이라서 상태명을 이렇게 해주면 좋을 것 같네요~
```suggestion
  const [user, setUser] = useState(null);
```